scripts/test_solidity_lsp.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python3.8
+
+import argparse
+import json
+import os
+import subprocess
+
+from typing import Callable, List, Union, Any
+from deepdiff import DeepDiff
+
+# {{{ JsonRpcProcess
```

Comment:
Why do you say ghetto namespacing? For simplicity it's all in one file. I can move it out though. The `{{{` and `}}}` are Vim folding markers.

Reply:
Just a joke :) I never really liked code folding and don't use it so I thought these were just your own markers. Inserting extra stuff into the code just for folding seems weird to me. I mean, it feels like you really need some semantic element there instead of comments.

I think it should be a separate module. I especially the utility code and test cases should be separated in my opinion.

Reply:
I still think it should have been a separate module, though maybe it's too late to change that for the MVP now.